clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchResponse.java

```diff
@@ -187,6 +132,31 @@ public OffsetFetchResponse(Struct struct) {
         this.error = struct.hasField(ERROR_CODE_KEY_NAME) ? Errors.forCode(struct.getShort(ERROR_CODE_KEY_NAME)) : topLevelError;
     }
 
+
+    private List<Struct> getTopicArray(Map<TopicPartition, PartitionData> responseData) {
```

Comment:
Looking at this method initially, I thought it should be static since we are passing `responseData` as a parameter instead of accessing the field, but then I noticed that we access the `struct` field. Is there a reason why it's done this way? Maybe we should remove the existing parameter or pass both as parameters.

Also, should the method name be `topicArray`?